core/capabilities/remote/executable/server.go

```diff
@@ -203,6 +203,11 @@ func (r *server) getMessageHash(msg *types.MessageBody) ([32]byte, error) {
 		return [32]byte{}, fmt.Errorf("failed to unmarshal capability request: %w", err)
 	}
 
+	// default to excluding StepDependency
```

Comment:
Could we add a comment here explaining the context?

An attribute called StepDependency is used to define a data dependency between steps, and not to provide input values; we should therefore disregard it when hashing the request

Reply:
Added!